### PR TITLE
Add missing env var for hosted mode debugging

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

**Comment by mprahl** [comment]
/override "ci/prow/test-e2e-grc-framework"

This only impacts the Kind GitHub workflow.

File: .github/workflows/kind.yml [hunk 1]
```diff
@@ -186,6 +186,8 @@ jobs:
     - name: Debug
       if: ${{ failure() }}
       working-directory: framework
+      env:
+        HOSTED: ${{ matrix.hosted }}
```
Comment (dhaiducek) [inline .github/workflows/kind.yml:190]: I'm not actually sure why we're not just putting this (and `deployOnHub`) at the `jobs` level?

https://github.com/stolostron/governance-policy-framework/blob/6531fc537ccab7b2f9b8bc6c082e5e6454c1c8d5/.github/workflows/kind.yml#L23-L25

**Review by dhaiducek — APPROVED** [review]
Thanks for cleaning this up! 😄 🧹

**Pull request merged**